snapcraft/storeapi/_store_client.py

```diff
@@ -172,6 +172,9 @@ def get_snap_revisions(self, snap_name, series=None, arch=None):
         except KeyError:
             raise errors.SnapNotFoundError(snap_name, series=series, arch=arch)
 
+        if not snap_id:
```

Comment:
should we do `is None` to contemplate the possibility of something else? Maybe not worth it, but it may be a valid `snap_id`

Reply:
I would like an explicit comparison to `None` too, we shouldn't make any assumptions about truthiness of valid `snap_id` s

Reply:
This is used [here](https://github.com/snapcore/snapcraft/blob/master/snapcraft/storeapi/_sca_client.py#L281). Surely there are no falsey values that would work in this case?

Reply:
Alright, I've updated the PR to compare to None.